cooked-validators/src/Cooked/Ltl.hs

```diff
@@ -271,19 +285,14 @@ interpLtl (Instr (Builtin b) f) = interpBuiltin b >>= interpLtl . f
 -- 'MonadModal' because there might be other possibilities to equip a monad with
 -- LTL modifications beside the method above.
 
--- | Introduce an LTL formula to be in effect from now on, modifying the
--- remainder of the computation.
-startLtl :: Ltl modification -> Staged (LtlOp modification builtin) ()
```

Comment:
Nice! I'm happy to see this removed; it is confusing for a user of the library :)